src/directwrite/mod.rs

```diff
@@ -333,3 +325,10 @@ impl TextAnalysisSourceMethods for TextAnalysisSourceData<'_> {
         dwrite::DWRITE_READING_DIRECTION_LEFT_TO_RIGHT
     }
 }
+
+impl From<HRESULT> for Error {
+    fn from(hresult: HRESULT) -> Self {
+        let message = format!("A DirectWrite rendering error occurred: {:#X}", hresult);
+        Error::PlatformError(message)
+    }
+}
```

Comment:
I feel like ideally we'd return proper errors here, but I can see why that would not be useful for downstream and unnecessarily complicated for crossfont, so it's probably fine.

Reply:
I was thinking about having PlatformError(SomeErrorType), but it seemed to me that there's literally no benefit in it and the only thing that matters is the message, so I've preserved the message and users will still know it, just removed complexity from using nested errors.